__tests__/app.test.js

```diff
@@ -87,3 +96,39 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: responds with an array of article objects, each containing correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles.length).toBeGreaterThan(0);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
+  test("404: responds with 'path not found' for invalid endpoint", () => {
+    return request(app)
+      .get("/api/articlez")
```

Comment:
You don't need to do this test again as you've already done it on line 40! That test that you did on line 40 should just be in it's own describe block, as this error is not specific to any particular endpoint, it's a general server error for an incorrect endpoint 👍